src/data/__tests__/calculate.test.ts

```diff
@@ -544,4 +545,86 @@ describe('calculate', () => {
       ).toEqual(defaultValue * 2 * 0.1)
     })
   })
+
+  describe('percentFullyVaccinated', () => {
+    const defaultScenario = {
+      ...testData(prepopulated['1person_15minCarRide']),
+      averageFullyVaccinatedMultiplier: 0.4,
+      percentFullyVaccinated: 40,
+      unvaccinatedPrevalenceRatio: null,
+    }
+    const defaultUnvaccinatedPrevalenceRatio = 1 / (0.4 * 0.4 + 0.6)
+
+    it.each`
+      theirVaccine
+      ${'vaccinated'}
+      ${'unvaccinated'}
+    `(
+      'Should decrease risk for lower vaccination rate when interacting with $theirVaccine people',
```

Comment:
Huh, this is right but feels counterintuitive to me.

Reply:
Yeah, it was counterintuitive to me too. I think the right way to think about it is that holding all the other data (case rates etc) static, a lower vaccination rate means that you have that same case rate with less vaccination which means the chance of getting COVID is lower (and vice versa of course)